pilot/pkg/networking/util/util.go

```diff
@@ -52,14 +52,19 @@ import (
 const (
 	// BlackHoleCluster to catch traffic from routes with unresolved clusters. Traffic arriving here goes nowhere.
 	BlackHoleCluster = "BlackHoleCluster"
+	// BlackHoleVirtualHost is the name of the virtual host used to block all traffic
+	BlackHoleVirtualHost = "block_all"
 	// BlackHoleRouteName is the name of the route that blocks all traffic.
-	BlackHoleRouteName = "block_all"
+	BlackHoleRouteName = "black_hole_route"
```

Comment:
How does changing the route name help in this scenario?

Reply:
FYI @rshriram it's still a WIP. Jacob might be in the middle of making it work.

Reply:
Combined with the PR for this https://github.com/istio/proxy/pull/2792 it works. I still need to review possible test cases I haven't covered.

Reply:
To add to this, it was the name of the virtual host that was `block_all` and `allow_any` and not the name of the route; the name of the route had no name. I added `BlackHoleVirtualHost` and `PassthroughVirtualHost` as variables with `block_all` and `allow_any`, respectively.